### PR TITLE
applies updated tags

### PR DESCRIPTION
**Pull request by m4sterbunny**
I seem to have only used 3 of the tags (did not use case or promise) -- please let me know where I missed opportunities

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,27 @@ Parameters as array by name (not by in)
 
 check application of proper nouns
 
+### Security
+
+Not sure if that declaration at line 20 works globally, i.e. may need to apply at each endpoint (see userLogin).
```
Comment (jehrlich) [inline README.md:20]: The Security setting becomes the default for all endpoints ... 
```
security:
  - authToken: [] 
```
UNLESS overridden for a particular endpoint.  `userLogin` should have no security, so 
`security: []`

File: src/openapi.yaml [hunk 2]
```diff
@@ -25,9 +25,9 @@ paths:
     post:
       operationId: userLogin
       tags:
-        - editor
-        - idempotent
+        - User
       security:
+        - authToken: []
```
Comment (jehrlich) [inline src/openapi.yaml:30]: This should be an empty array, so that the user can log in.

**Review by jehrlich — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (m4sterbunny) [reply README.md:20]: that makes sense -- just the swagger linter did not like the existing format, will see what I can figure out and return login to none

File: src/openapi.yaml (hunk 2 above)
Comment (m4sterbunny) [reply src/openapi.yaml:30]: figured out a way to give it empty array that swagger editor does not moan about -- will have to see if Redoc.ly likes it too

**Pull request merged**